### DISCUSSION TITLE
New issue labels (2021)

### DISCUSSION THREAD

**Discussion by Jonta** [AnySoftKeyboard/AnySoftKeyboard#3023] [Ideas]
### Background

- [I've labeled](https://github.com/AnySoftKeyboard/AnySoftKeyboard/discussions/2789#discussioncomment-1277235) the first 11+ pages of open issues now
- I thought I was going to get through them all before making this thread
- But I'm going to prioritise other things for a while
- Don't know when I'll be back to labeling the rest
- ASK should have this thread sooner rather than later

### Documentation

https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels

### Question

What's a `Library-issue`s?

### General suggestions

- Let people who label issues also change issue titles
- Explain labels. Example: https://github.com/microsoft/terminal/labels

### Label suggestions

- I can go into depth on any y'all are iffy about
- I don't know how readily these are going to be accepted
- Therefore I didn't put a _lot_ of work into these
- I kept a running tally of ones I thought might be nice to have
- There's still polishing to be done, if it seems okay so far

* Emoji
* Layouts
  * T9
  * QWERTY
  * Dvorak
  * Halmak
  * QWERZ
  * Colemak
  * Bépo
* Languages / Language-specific (Only happens with a language)
  * And 1 per language E.g. "Lang::Polish" (With prefix, to make that block easy to see in the list of tags)
    * Bengali
    * Czech
    * English
    * German
    * Greek
    * Hebrew
    * Hindi
    * Icelandic
    * Indonesian
    * Kurdish
    * Macedonian (?)
    * Norwegian
    * Persian (?)
    * Portuguese
    * Slovak
    * Slovenian
    * Spanish
    * Swedish
    * Swiss
    * Tamil
    * Turkish
  * Language adding request
* Swiping
* Stores
  * Fdroid
  * Google Play Store
  * Aurora
* Animation
* Regression
* Password
* Spelling suggestions (Dictionaries?)
* Abbreviations
* Punctuation
* Clipboard
* Tablet
* Phone
* Laptop
* Graphics (logos and more)
* Themes
* Decoration
* Appearance
* Sounds
* Security
* Unwanted characters appearing
* Wanted characters not appearing
* Blocking and the other Firefox one
* Freeze
* Spelling
* Grammar
* Special keys (e.g. Tab)
* Documentation
* Beta
* Releases? Versions?
* a11y
* dupeme ("I'm confident this duplicate is an issue, but I don't remember which issue it's duplicating")
* Bluetooth
* External stuff (e.g. F-droid)
* Bad issue title
* Low effort put into issue (not following format, spelling, wall of text)
* Code itself, programming, building ASK oneself
* Right-to-left, left-to-right
* Light mode/dark mode
* Backup
* Severity & Priority (Mozilla style. S1, S2 etc & P1, P2 etc)
* Devices / Manufacturers
  * Xiamo
  * Motorola
  * Samsung
  * Fairphone
* Curved display
* Could not reproduce
* Could reproduce
* Apps (Doesn't work in GDocs, e.g.)
* Legal / License

**Comment by Jonta** [comment]
@menny Half-year update? =)

**Comment by aslanenver** [comment]
Turkish keyboard layout is not like in gboard.how to fix. will there be a new update![Screenshot_20221028-085620_GitHub~01.png](https://user-images.githubusercontent.com/75860223/198513931-444c6df2-ebd5-4f9d-a70e-9203fee6f6c2.png)

**Reply by Jonta** [reply]
@aslanenver
- Your comment is off topic in this thread
- Please create an issue instead. :)

**Reply by aslanenver** [reply]
> @aslanenver
> - Your comment is off topic in this thread
> - Please create an issue instead. :)

Sorry. :)
